aspnetcore/diagnostics/code-analysis.md

```diff
@@ -32,6 +32,11 @@ uid: diagnostics/code-analysis
 | [ASP0017](xref:diagnostics/asp0017) | Non-breaking             | Invalid route pattern |
 | [ASP0018](xref:diagnostics/asp0018) | Non-breaking             | Unused route parameter |
 | [ASP0019](xref:diagnostics/asp0019) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
+| [ASP0020](xref:diagnostics/asp0020) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
+| [ASP0021](xref:diagnostics/asp0021) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
+| [ASP0022](xref:diagnostics/asp0022) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
+| [ASP0023](xref:diagnostics/asp0023) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
+| [ASP0024](xref:diagnostics/asp0024) | Non-breaking             | Suggest using IHeaderDictionary.Append or the indexer |
```

Comment:
The description column needs to be updated for each diagnostic. But you might consider removing the table altogether and relying on the TOC for the list. This is a lot of duplication.